### PR TITLE
chore: editing apollo client config #132

### PR DESCRIPTION
**Pull request by ilya-trifonov**
affects: @amplicode/codegen

### CODE REVIEW

File: example-app/src/app/App.test.tsx [hunk 1]
```diff
@@ -48,9 +48,7 @@ it("renders without crashing", () => {
```
Comment (vyacheslav-pushkin) [inline example-app/src/app/App.test.tsx:48]: Requirement from the ticket:

> This field should be sorted out and not shown on screens.

**Review by vyacheslav-pushkin — CHANGES_REQUESTED** [review]

File: example-app/src/app/screens/owner/OwnerList.tsx [hunk 2]
```diff
@@ -166,7 +166,7 @@ export const OwnerList = observer(({ onSelect }: EntityListScreenProps) => {
 const Fields = ({ entity }: { entity: any }) => (
   <>
     {Object.keys(entity)
-      .filter(p => p !== "id" && entity[p] != null)
+      .filter(p => p !== "id" && p !== "__typename" && entity[p] != null)
```
Comment (vyacheslav-pushkin) [inline example-app/src/app/screens/owner/OwnerList.tsx:169]: We should extract it into a separate function.

File: example-app/src/app/screens/owner/OwnerList.tsx [hunk 3]
```diff
@@ -166,7 +166,7 @@ export const OwnerList = observer(({ onSelect }: EntityListScreenProps) => {
...
       .map(p => (
```
Comment (vyacheslav-pushkin) [inline example-app/src/app/screens/owner/OwnerList.tsx:170]: Looks like it will still be shown in editor.

**Review by vyacheslav-pushkin — CHANGES_REQUESTED** [review]

File: example-app/src/app/screens/owner/OwnerList.tsx (hunk 2 above)
Comment (ilya-trifonov) [reply example-app/src/app/screens/owner/OwnerList.tsx:169]: @vyacheslav-pushkin  Should I do it like this?
.filter(p => p !== "id" && entity[p] != null)
.filter(p => p !== "__typename")

Comment (vyacheslav-pushkin) [reply example-app/src/app/screens/owner/OwnerList.tsx:169]:
```
.filter(getDisplayedAttributes)

// Outside the component
function getDisplayedAttributes(attribute: string) {
  return attribute !== "id" && etc.
}
```

**Review by vyacheslav-pushkin — APPROVED** [review]

**Comment by vyacheslav-pushkin** [comment]
@ilya-trifonov Please resolve conflicts and merge

**Pull request merged**